rocAL/rocAL/source/api/rocal_api_augmentation.cpp

```diff
@@ -567,6 +567,55 @@ rocalResize(
     return output;
 }
 
+RocalImage  ROCAL_API_CALL
+rocalResizeMirrorNormalize(
+            RocalContext p_context,
+            RocalImage p_input,
+            unsigned dest_width,
+            unsigned dest_height,
+            std::vector<float> &mean,
+            std::vector<float> &std_dev,
+            bool is_output,
+            RocalIntParam p_mirror)
+{
+    if(!p_context || !p_input || dest_width == 0 || dest_height == 0 )
+        THROW("Null values passed as input")
+    Image* output = nullptr;
+    auto context = static_cast<Context*>(p_context);
+    auto input = static_cast<Image*>(p_input);
+    auto mirror = static_cast<IntParam *>(p_mirror);
+    for(unsigned i = 0; i < mean.size(); i++) {
```

Comment:
This is not a good idea to reset the mean and std here. If we need to use 0 and 1, that has to be passed to the API from caller

Reply:
We are storing the std and mean from the user in the Pipeline object and using that for normalization in rocALcopyOutTensor. So we are passing 0 and 1 to RPP kernel to overcome this. This is adapted from CMN [node](https://github.com/SundarRajan28/MIVisionX/blob/RMN_dev/rocAL/rocAL/source/api/rocal_api_augmentation.cpp#L1716) where we pass 0 and 1 to CMN RPP kernel.